runtime/onert/core/include/ir/OperationIndexSet.h

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2018 Samsung Electronics Co., Ltd. All Rights Reserved
```

Comment:
`2020` ?

Reply:
This file is from `OperationIndexList.h`
I don't know how we write the year of copyright, if a file's name is modified.